HOW_TO_RUN.md

```diff
@@ -1,52 +1,60 @@
-Thi
+# starbake
 
-## Install 
+## Prerequisites
+
+Before installing starbake, ensure the following minimum versions are installed on your system:
+
+- jdk: 11 or higher
+- python: 3.8 or higher
+
+## Install
 
 1. Install Starlake
-    `SL_VERSION=1.0.0-SNAPSHOT sh /Users/elarib/Work/Starlake/starlake/distrib/starlake.sh`
-2. Generate dummy files
-   `pip install faker`
-   `python _scripts/dummy_data_generator.py`
+`sh <(curl https://raw.githubusercontent.com/starlake-ai/starlake/master/distrib/setup.sh) --target=.`
+2. Create a virtual environment (optional)
+   `python3 -m pip install virtualenv`
+   `python3 -m venv .venv`
+3. Activate the virtual environment (optional)
+   `source .venv/bin/activate`
+4. Generate dummy files
+   `python3 -m pip install -r _scripts/requirements.txt`
+   `python3 _scripts/dummy_data_generator.py`
 
 We're good to go
 
-
-## Run Starlake 
+## Run Starlake
 
 1. Import from incoming to pending
-`sh starbake.sh import`
-
+`./starlake import`
 
 2. Load data to bigquery
-`sh starbake.sh load`
+`./starlake load`
 
 3. Run the transformation in order
-```
-sh starbake.sh transform --name Customers.CustomerLifetimeValue 
-sh starbake.sh transform --name Customers.HighValueCustomers 
-
-sh starbake.sh transform --name Products.ProductProfitability 
-sh starbake.sh transform --name Products.MostProfitableProducts 
 
-sh starbake.sh transform --name Products.ProductPerformance 
-
-sh starbake.sh transform --name Products.TopSellingProducts 
-sh starbake.sh transform --name Products.TopSellingProfitableProducts 
-```
+```bash
```

Comment:
Is it better maybe to change those command to 
./starlake transform --name Products.TopSellingProfitableProducts --recursive
  WDYT ?

Reply:
I added a step to recursively execute 2 transformations which are enough to run all the transformations, is this ok for you?